tool/readme_scripts/readme_scripts_runner.dart

```diff
@@ -0,0 +1,184 @@
+// ignore_for_file: avoid_print
```

Comment:
Elsewhere we put the `ignore_for_file` directives after the copyright block and a blank line (e.g., `download_preplanned_map_area_sample.dart`). It's probably best to have the copyright block always treated with reverence.